.teamcity/Gradle_Check/configurations/GradleBuildConfigurationDefaults.kt

```diff
@@ -27,6 +27,12 @@ fun shouldBeSkipped(subProject: GradleSubproject, testConfig: TestCoverage): Boo
     return testConfig.os.ignoredSubprojects.contains(subProject.name)
 }
 
+val cleanDaemonsBeforePerformanceTest = """
+    free -m
+    ps aux | egrep 'Gradle(Daemon|Worker)' | awk '{print ${'$'}2}' | xargs kill -9
```

Comment:
I wonder how safe `kill -9` is. It doesn't give the Gradle Daemon any chance to clean up. Can that be a problem?

Reply:
It's why I recommended calling `./gradlew --stop` first

Reply:
Hmmmm... though I'm not 100% sure, but on my MBP I have a simple alias to kill all Gradle processes,

```
# zhb @ bs-MacBook-Pro in ~/Projects/tmp/options [16:18:07] 
$ which kds
kds: aliased to ps aux | grep Gradle | awk '{print $2}' | xargs kill -9
```

I use it very often and never see weird things.